### PR TITLE
[maps feature]: shows some locations using the maps api

### PR DESCRIPTION
**Pull request by kiprot**
- added a museums.csv file with details of some locations
- added a servlet to create a json string of location objects
- added a function in script.js to create map
- link: https://kiprotich-step2020.uc.r.appspot.com/dinosaur.html

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -104,4 +104,28 @@ function fetchBlobstoreUrlAndShowForm() {
...
         messageForm.classList.remove('hidden');
     });
+}
+
+/** Creates a map centered in Manhattan NY with some museum locations. */
+function createMap() {
+    fetch('/museum-locations').then(response => response.json()).then((museumLocation) => {
+        const map = new google.maps.Map(
+            document.getElementById('map'),
+            {center: {lat: 40.7831, lng: -73.9712}, zoom: 12} );
```
Comment (addarsh-google) [inline portfolio/src/main/webapp/script.js:114]: i think you should store this as a constant so its easy to understand that it's the Manhattan NY coords.

File: portfolio/src/main/java/com/google/sps/data/MuseumLocations.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private String name;
+    private double lat;
+    private double lng;
+    private int layer;
+
+    public MuseumLocations(String name, double lat, double lng, int layer) {
+        this.name = name;
+        this.lat = lat;
+        this.lng = lng;
+        this.layer = layer;
```
Comment (addarsh-google) [inline portfolio/src/main/java/com/google/sps/data/MuseumLocations.java:28]: what does layering do?

**Review by addarsh-google — APPROVED** [review]
Well done! I really liked the implementation on your site.

File: portfolio/src/main/java/com/google/sps/data/MuseumLocations.java (hunk 2 above)
Comment (kiprot) [reply portfolio/src/main/java/com/google/sps/data/MuseumLocations.java:28]: it positions the map location pins in such a way such that if there are 2 locations that are near each other, one pin is "layered" on top of another in the order that is defined

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (kiprot) [reply portfolio/src/main/webapp/script.js:114]: ok, noted

**Pull request merged**